src/enclave/rpccontext.h

```diff
@@ -17,6 +17,7 @@ namespace enclave
   {
     size_t client_session_id = InvalidSessionId;
     std::vector<uint8_t> caller_cert = {};
+    bool is_forwarded = false;
```

Comment:
The name of this variable is perhaps a little confusing? Out of context (no pun intended), I would expect this to be `true` if this is a forwarded command, i.e. a `SessionContext` on the primary node for a forwarded command. Would `bool to_forward = false;` be clearer?

Reply:
forward = false; even?